tools/sloths/src/components/switchers/SoundSwitcher.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="sound-switcher">
+    <input
+      class="sound-switcher__input"
+      type="radio"
+      v-for="(sound, i) in sounds"
+      :key="`sound-${i}`"
+      :value="sound"
+      v-model="currSound"
+      :name="`sound-${sound}`"
+      :id="`sound-${sound}`"
+    />
+
+    <label
+      class="sound-switcher__label"
+      :for="`sound-${checkSound}`"
+      :class="`sound-switcher__label_${currTheme}-${checkSound}`"
+      :title="$t('header.switchers.sound')"
+      v-shortkey="['ctrl', '1']"
+      @shortkey="setSoundValue(`${checkSound}`)"
+    ></label>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { mapWritableState } from 'pinia';
+
+import useAudioOn from '../../stores/audio-on';
+import themeProp from '../../stores/theme';
+
+export default defineComponent({
+  name: 'ThemeSwitcher',
+
+  data(): { currSound: string; sounds: string[] } {
+    return {
+      currSound: '',
+      sounds: ['on', 'off'],
+    };
+  },
+
+  computed: {
+    ...mapWritableState(useAudioOn, ['isAudioOn']),
+    ...mapWritableState(themeProp, ['currTheme']),
+    checkSound(): string {
```

Comment:
Names of computed properties shouldn't be verbs

Reply:
done
rename all computed properties with verbs